### ISSUE TITLE
Copyright Header

### ISSUE THREAD

**Issue by vertexclique** [assaultjs/assaultjs#11]
Should we change it to smt like `Copyright AssaultJS Project 2015` or everybody should put their own field?

**Comment by jesusprubio** [comment]
Yep this is my idea, a lo of free projects work just like that. Everyone who touches a file should add its name there.
Moreover I've licensed each file because the FSF recommends it and in this way everyone could license his modules like he wanted. As @jpenalbae commented here(https://github.com/assaultjs/assaultjs/issues/6) It's important to be able of using libraries with different licenses in our framework.
